### ISSUE TITLE
Non-scout delivery causing errors for MIP-dna cases

### ISSUE THREAD

**Issue by Vince-janv** [Clinical-Genomics/cg#1397]
## Description

cg attempts to upload MIP-DNA cases to scout even if they do not have scout as data delivery option. Similarly cg attempts to upload delivery reports for the same cases to scout.

## Suggested solution

A quick look seems to indicate that [this query](https://github.com/Clinical-Genomics/cg/blob/7a2cc82e09f837d2d68ce50cab577d8997b11563/cg/store/api/status.py#L652-L673) does not include a filter for `models.Family.data_delivery`. Maybe it should?

## This can be closed when

MIP-dna cases can be delivered without error, when a data_analysis other than scout is chosen.

## Blocked by

*If there are any blocking issues/prs/things in this or other repos. Please link to them.*

**Comment by Vince-janv** [comment]
I'm assuming a delivery report should still be created and sent to caesar with the other case files for these cases. Should that be a separate issue @henrikstranneheim?

**Comment by henrikstranneheim** [comment]
Yes, that is a good point. We should include it in the deliver command. @ivadym this would also support your suggestion of separating the creation and storing of the delivery report from the uploading to scout.

**Comment by henrikstranneheim** [comment]
Added to refinement session tomorrow
